### PR TITLE
Fixed message rendering issues on chat page.

### PR DESCRIPTION
**Pull request by code-with-om**
Fixed- [issue-148](https://github.com/OpenNyAI/Jugalbandi-Manager/issues/148#issuecomment-2244429050)
- Updated the `create_message` functionality to insert messages in JSON format in the JBMessage Table.
- Updated the frontend code of the `loadMessage` function in chat.tsx to render each component properly.

### CODE REVIEW

File: channel/src/crud.py [hunk 1]
```diff
@@ -35,7 +33,7 @@ async def create_message(
                     turn_id=turn_id,
                     message_type=message_type,
                     is_user_sent=is_user_sent,
-                    message=message,
+                    message=json.loads(message),
```
Comment (shreypandey) [inline channel/src/crud.py:36]: @code-with-om This method expects a dict. Can we add `json.loads` while calling the `create_message`?

File: flow/src/crud.py [hunk 2]
```diff
@@ -192,7 +192,8 @@ async def create_message(
                     turn_id=turn_id,
                     message_type=message_type,
                     is_user_sent=is_user_sent,
-                    message=message,
+                    message=json.loads(message),
```
Comment (shreypandey) [inline flow/src/crud.py:195]: @code-with-om This method expects a dict. Can we add json.loads while calling the create_message?

File: channel/src/crud.py (hunk 1 above)
Comment (code-with-om) [reply channel/src/crud.py]: This is done

File: flow/src/crud.py (hunk 2 above)
Comment (code-with-om) [reply flow/src/crud.py]: This is done

**Review by shreypandey — APPROVED** [review]
Added changes for rendering messages in frontend

**Pull request merged**